src/praisonai/praisonai/framework_adapters/praisonai_adapter.py

```
@@ -26,24 +26,123 @@ def is_available(self) -> bool:
+            agent_tasks = details.get('tasks', {})
+            if not agent_tasks:
+                # Auto-generate a task
+                task_description = details.get('instructions') or backstory_filled
+                task = PraisonTask(
+                    description=task_description,
+                    expected_output="Complete the assigned task successfully.",
+                    agent=agent,
+                )
+                tasks.append(task)
+            else:
+                for task_name, task_details in agent_tasks.items():
```

Ruff (0.15.12)

[warning] Loop control variable `task_name` not used within loop body

Rename unused `task_name` to `_task_name`

(B007)